doc/service_interface.md

```diff
@@ -0,0 +1,70 @@
+# Radar Service Interface
+
+The Service Interface is a simple HTTP-based interface for services to query radar.
+
+## Service Endpoint
+
+- http(s)://hostname:port/radar/service
+
+## GET
+
+E.g. `http://localhost:8000/radar/service?to=status:/segment/foobar`
+
+### Querystring Parameters
+- `to` : the full resource scope, e.g. `status:/segment/foobar`
+
+This is a simplified case of the POST endpoint for Radar resources supporting `get` operations.
+
+### Response
+A `get server message` depending on the resource type, e.g. for a Status:
+
+```json
+{
+  "op": "get",
+  "to": "status:/account/clients/1452556889570",
+  "value": {
+    "1452556889570": "sdf"
+  }
+}
+```
+
+or for a Presence:
+
+```json
+{
+  "op": "get",
+  "to": "presence:/account/box1",
+  "value": {
+    "1452560641403": 2
```

Comment:
I refer to https://github.com/zendesk/radar/blob/jden/service_interface/core/lib/resources/presence/presence_manager.js#L336  which is what get should respond to (https://github.com/zendesk/radar/blob/jden/service_interface/core/lib/resources/presence/index.js#L212)

the old version is not very useful anymore.